pipelines/run-pipelines-on-apache-airflow/README.md

```diff
@@ -0,0 +1,332 @@
+
+1. Enter the following values for the `SimpleHttpOperator` properties:
+   
+   - `endpoint` -> `/repos/elyra-ai/examples/content/pipelines/run-pipelines-on-apache-airflow/resources/command.txt`
+      - Since this property is implicity required in the operator specification file, the pipeline editor displays a red bar and enforces the constraint. 
+      
+      ![A required property](doc/images/required-property.png)
+     
+   - `method` -> `GET`
+   - `data` -> `{"ref": "master"}`
+      - This information tells the GitHub API which branch to use when returning the file contents
+   - `headers` -> `{"Accept": "Accept:application/vnd.github.v3.raw"}`
+      - This tells the API what format the returned data should be
+      - In this case, we want the raw GitHub file
+   - `xcom_push` -> check the checkbox for `True`
+      - This property indicates to Airflow whether we want to pass on the output of this component (in this case, the file contents of our requested file) to be accessed by later nodes in the pipeline
+   - `http_conn_id` -> `http_github`
+      - This property tells the Airflow instance which `Connection` id it will use as the API base URL
+      - This property was configured in the above section, _Configure the runtime configuration_ (link??)
+
+
+   ![Configure request node](doc/images/configure-request-node.png)
+
+
+1. Open the properties of the '`BashOperator`' node. The [specification for the underlying component looks as follows](https://raw.githubusercontent.com/elyra-ai/examples/master/pipelines/run-pipelines-on-apache-airflow/components/bash_operator.py):
+  ```
+  class BashOperator(BaseOperator):
+    """
+    Execute a Bash script, command or set of commands.
+
+    ...
+
+    :param bash_command: The command, set of commands or reference to a
+        bash script (must be '.sh') to be executed. (templated)
+    :type bash_command: str
+    :param xcom_push: If xcom_push is True, the last line written to stdout
+        will also be pushed to an XCom when the bash command completes.
+    :type xcom_push: bool
+    :param env: If env is not None, it must be a mapping that defines the
+        environment variables for the new process; these are used instead
+        of inheriting the current process environment, which is the default
+        behavior. (templated)
+    :type env: dict
+    :param output_encoding: Output encoding of bash command
+    :type output_encoding: str
+   ...
+   ```
+
+In Apache Airflow, the output of a component can be used as a property value for any downstream node. (A downstream node is a node that is connected to and executed after the node in question). The pipeline editor renders a selector widget for each property that allows you to choose between two options as a value:
```

Comment:
Missing indent breaks numbered list rendering (list items "The contents of the file ..." and ff)
 
```suggestion
   In Apache Airflow, the output of a component can be used as a property value for any downstream node. (A downstream node is a node that is connected to and executed after the node in question). The pipeline editor renders a selector widget for each property that allows you to choose between two options as a value:
```